src/routes/role.js

```diff
@@ -0,0 +1,179 @@
+import { Router } from 'express';
+import { body, param, validationResult } from 'express-validator';
+
+import { authenticate } from '../modules/auth';
+import models from '../modules/database';
+import loggers from '../modules/logging';
+
+const router = Router();
+const log = loggers('role');
+const { Role } = models;
+
+/**
+ * GET Role Info
+ * @param roleId int
+ */
+router.get(
+  '/:id(\\d+)',
+  authenticate(),
+  [
+    param('id')
+      .isNumeric()
+      .isInt({ min: 1 })
+      .toInt()
+  ],
+  async (req, res) => {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
+    const { id } = req.params;
+
+    log.info(`request for role id ${id}`);
+    try {
+      const result = await Role.findOne({
+        where: {
+          id
+        }
+      });
+
+      if (!result || result.length === 0) {
+        return res.status(204).end();
+      }
+
+      res.type('application/json');
+      res.json(result);
+    } catch (error) {
+      log.error(error.message);
+      res.status(500).send(error.message);
+    }
+  }
+);
+/**
+ * PUT Update Role
```

Comment:
I'm going to start on a branch that will add something like an `ensureRole(name)` middleware so that these routes can be protected, otherwise this wouldn't be suitable for production. I'll make a PR as soon as it's ready.

Reply:
OK, if you don't mind adding the `ensureRole('Administrator')` to these I'd appreciate it, forgot to do so in my PR.